scripts/firedrake-install

```diff
@@ -1662,6 +1662,12 @@ else:
         build_update_script()
         os.execv(sys.executable, [sys.executable, "../bin/firedrake-update", "--no-update-script"] + sys.argv[1:])
 
+    # Pre-install requested packages
+    if args.pip_packages is not None:
+        for package in args.pip_packages:
+            log.info("Pip installing %s to venv" % package)
+            run_pip_install(package.split())
```

Comment:
Why is there code in this PR?

Reply:
Prior to this PR the code to install additional packages with pip was only present in the "install" part of the install script, not the "update" part of the script. Running `firedrake-update --pip-install scipy` would have ignored the scipy package.